fendermint/storage/src/im.rs

```diff
@@ -183,6 +203,48 @@ where
     }
 }
 
+struct KVIter<'a, S: KVStore, K, V> {
+    items: Vec<(&'a S::Repr, &'a S::Repr)>,
+    next: usize,
```

Comment:
maybe `curr_size` ?

Reply:
What I wanted to express here is the next index that will be returned from the `Vec`. The vec isn't being consumed, its size stays the same.